dvc/commands/experiments/ls.py

```diff
@@ -18,15 +18,13 @@ def run(self):
         )
         tags = self.repo.scm.describe(exps)
         remained = {baseline for baseline, tag in tags.items() if tag is None}
-        base = "refs/heads/"
+        base = "refs/heads"
```

Comment:
@pmrowla @karajan1001 it seems to be a separate bug here. In the `describe` we have a check for this:

https://github.com/iterative/scmrepo/blob/main/src/scmrepo/git/__init__.py#L452-L459

It seems to be related to the order in which heads are processed for a commit that is head of multiple branches? But I'm not sure (we need a comment in the test, and overall ideally test should be split into a few with proper semantic names, it too loaded. I mean the `describe` test).

could you please confirm?

Reply:
Can we move all that logic into `dulwich` implementation?

Reply:
The extra `HEAD` logic in the scmrepo describe implementation is so that we display the current checked out branch when possible (i.e. if you have multiple branches pointing to the same commit SHA, we will prefer the checked out one which is set in `HEAD`). It's not specific to dulwich.

Reply:
thanks @pmrowla. yep, my question here I guess is - since we are already implementing this in the dulwich wrapper (`_describe` or something), can we move it there ... anyways, minor thing.